src/main/java/com/google/starfish/servlets/Utils.java

```diff
@@ -0,0 +1,29 @@
+package com.google.starfish.servlets;
+
+import com.google.gson.Gson;
+import com.google.starfish.services.FavoriteNoteService.Recency;
+
+public class Utils {
+
+  /** Converts a 2D array to JSON */
+  public static String convert2DArrayToJSON(Object[][] arr) {
+    Gson gson = new Gson();
+    return gson.toJson(arr);
+  }
+
+  /** Finds key of Recency enum by string value */
+  public static Recency findRecencyByString(String recency) {
```

Comment:
I think this looks fine-- I like that you extracted these out to a Utils file!

Reply:
Thanks! Yeah we talked about that tradeoff between complexity and consistency offline but for such small enums it really isn't too big of a deal.